src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
```

Comment:
Consider using naming conventions to name your constants https://github.com/pdabizha/js_task-transportation-on-vacation/blob/develop/checklist.md